packages/core/components/FileMetadataSearchBar/test/FileMetadataSearchBar.test.tsx

```diff
@@ -116,26 +113,30 @@ describe("<FileMetadataSearchBar />", () => {
         expect(getByText("Sun Aug 30 2020")).to.not.be.empty;
```

Comment:
could we add checks of 2020-08-30, 2020-08-31, 2020-09-01, 2021-03-03,2021-03-04, 2021-03-05?  Just to pin down the edges being what we want.

Reply:
I agree that it would be nice to have some additional test coverage around this logic. There are some existing patterns in the codebase related to test parameterization to avoid copy/pasting large tests just to change some inputs/expectations, let me know if you want a pointer to one of those.

Reply:
Parameterized this test and added a few cases to it